feature-libs/user/account/components/verification-token-form/verification-token-form-component.service.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import {
+  UntypedFormControl,
+  UntypedFormGroup,
+  Validators,
+} from '@angular/forms';
+import {
+  AuthService,
+  GlobalMessageService,
+  GlobalMessageType,
+} from '@spartacus/core';
+import { VerificationTokenFacade } from '@spartacus/user/account/root';
+import { BehaviorSubject, from } from 'rxjs';
+import { tap, withLatestFrom } from 'rxjs/operators';
+
+const globalMsgShowTime: number = 10000;
+@Injectable()
+export class VerificationTokenFormComponentService {
+  constructor(
+    protected auth: AuthService,
```

Comment:
Our recommendation is to use `inject` (https://angular.io/api/core/inject) method instead of passing services into constructor params.
Example:
```
protected auth = inject(AuthService);
```

etc.

Can you refactor your code and use such convention?

Reply:
fixed